app/screens/create_or_edit_channel/channel_info_form.tsx

```diff
@@ -201,29 +256,29 @@ export default function ChannelInfoForm({
             currentPurpose !== oldPurpose || currentHeader !== oldHeader;
     };
 
-    const onDisplayNameChangeText = (text: string) => {
+    const onDisplayNameChangeText = useCallback((text: string) => {
         onDisplayNameChange(text);
         if (editing) {
             enableRightButton(canUpdate(text, purpose, header));
             return;
         }
         const displayNameExists = text?.length >= 2;
         enableRightButton(displayNameExists);
-    };
+    }, [purpose, header]);
```

Comment:
Maybe it is not relevant in this case, but these are dependant on:
- `onDisplayNameChange` and `enableRightButton`, props that may or may not change between renders
- `canUpdate`, which depends on the `oldX` props, that may or may not change between renders.

Reply:
`onDisplayNameChange` is memoized with and comes in as a prop that is produced via a `useState` in the parent component.
`enableRightButton` is also memoized via the parent component. 

`canUpdate` does depend on `oldX` props which are created when user is editing a channel. I guess they could be updated if the user is editing a channel, and they or someone else edits the channel simultaneously via a differnent client and a websocket event triggers the change??  What are the concerns if I do add these as dependancies?